bitbots_vision/src/bitbots_vision/vision_modules/field_boundary.py

```diff
@@ -10,10 +10,21 @@
 
 
 class FieldBoundaryDetector(object):
+    """
+    This is the abctract class for the field boundary detector.
+    The task of the field boundary detector module is the localisation of the edges of the field.
+    It returns a list of points that form this so called field boundary.
+    It requires the ColorDetector to find the green pixels that resemble the field in the picture.
+    The green pixels of the ordinary FieldBoundary are found by traversing the picture from left to right and from top to bottom in steps of a given length.
```

Comment:
```suggestion
    The green pixels of the ordinary FieldBoundary are found by traversing the picture column wise in steps of a given length.
```
This doesn't make it clear we're skipping x colums inbetween, but i find colum wise sounds better and describes better the process